vendor/ember-power-select-bootstrap.css

```diff
@@ -196,8 +196,13 @@
   -webkit-user-select: none; }
   .ember-power-select-options[role="listbox"] {
     overflow-y: auto;
-    /* in firefox in windows this can cause a word-break issue. Try `overflow-y: scroll` if that happens */
     max-height: 14em; }
+  @supports (-webkit-overflow-scrolling: touch) {
```

Comment:
Files in /vendor are autogenerated, but don't worry. They'll be overwritten the next time I publish a version.